Tribler/community/chant/orm.py

```diff
@@ -0,0 +1,53 @@
+
+from datetime import datetime
+from pony import orm
+from Tribler.community.chant.MDPackXDR import serialize_metadata_gossip
+
+db = orm.Database()
+
+class Peer(db.Entity):
+    id               = orm.PrimaryKey(int, auto=True)
+    public_key       = orm.Required(buffer)
+    trusted          = orm.Optional(bool)
+    votes            = orm.Optional(int)
+    update_timestamp = orm.Required(datetime)
+
+class MetadataGossip(db.Entity):
+    id                 = orm.PrimaryKey(int, auto=True)
+    type               = orm.Required(int)
+    sig                = orm.Optional(buffer)
```

Comment:
I would spell this out, so just use _signature_

Reply:
Ok, I'll change it.